src/cb_admin/pages/Dashboard.js

```diff
@@ -50,9 +50,10 @@ export default class Dashboard extends React.Component {
   }
 
   componentDidMount() {
-    CommunityBusiness.get({ fields: ['name'] })
+    CommunityBusiness.update() // used to check cookie permissions
+      .then(CommunityBusiness.get({ fields: ['name'] }))
```

Comment:
Needs to be a function

Reply:
What do you mean?

Reply:
The value passed to `then` must be a function. You're passing a `Promise`.

Reply:
Oh yeah...

Weird that this works for the `Activities` page